sdgym/synthesizers/generate.py

```diff
@@ -19,6 +20,7 @@
     'TVAESynthesizer': TVAESynthesizer,
     'PARSynthesizer': PARSynthesizer,
     'HMASynthesizer': HMASynthesizer,
+    'RealTabFormerSynthesizer': RealTabFormerSynthesizer,
```

Comment:
Changes to make `create_sdv_synthesizer_variant` work with `RealTabFormerSynthesizer`.